config/config.go

```diff
@@ -25,28 +25,36 @@ type BombSquadConfig struct {
 	SuppressedMetrics map[string]BombSquadLabelConfig
 }
 
-func ReadBombSquadConfig(c Configurator) BombSquadConfig {
-	b := c.Read()
+func ReadBombSquadConfig(c Configurator) (BombSquadConfig, error) {
+	b, err := c.Read()
+	if err != nil {
+		return BombSquadConfig{}, fmt.Errorf("Failed to read Bomb Squad config: %s\n", err)
+	}
+
 	bscfg := BombSquadConfig{}
-	err := yaml.Unmarshal(b, &bscfg)
+	err = yaml.Unmarshal(b, &bscfg)
 	if err != nil {
-		log.Fatalf("Couldn't unmarshal into config.BombSquadConfig: %s\n", err)
+		return BombSquadConfig{}, fmt.Errorf("Couldn't unmarshal into config.BombSquadConfig: %s\n", err)
```

Comment:
Don't add \n to errors. Also, errors.Wrap from github.com/pkg/errors could be pretty nice for this pattern.

Reply:
Looks like you do this in a few places.

Reply:
Ah, yea, didn't consider how they would accumulate as the errors bubble up. Force of habit for prints, not great in the error itself. :-)

Reply:
errors.Wrap would be nice for sure. I'll do that in later PR, as this one's gotten kinda hairy as it is. :-)